### DISCUSSION TITLE
More recent RPMs

### DISCUSSION THREAD

**Discussion by vrpascuzzi** [AdaptiveCpp/AdaptiveCpp#488] [Q&A] [answered]
Thanks for this nice work, but can you produce some more recent RPMs? I am looking here:
http://repo.urz.uni-heidelberg.de/sycl/rpm/centos7/

and, although the modified date shows 11 February 2021, the latest builds are from October 2020. These RPMs greatly reduce the amount of work from the users' side.

Cheers!

**Comment by illuhad** [comment, marked as answer]
We are already working on it - we are in the process of switching to a new process to build packages which will hopefully allow us to publish more robust packages in the future.
@sbalint98 can tell you more.

**Reply by vrpascuzzi** [reply]
After a bit of digging, I do see:
http://repo.urz.uni-heidelberg.de/sycl/test-plugin/rpm/centos7/

Are these OK to use? I'm not sure what "test-plugin" refers to.

**Reply by illuhad** [reply]
It refers to packages that are constructed around the new plugin architecture of the hipSYCL runtime that was introduced very recently. These packages are not yet officially released. @sbalint98, can you comment on the expected stability of those?

**Reply by vrpascuzzi** [reply]
They seem to work, at least for my purposes.

Looking forward to the next release.

**Reply by sbalint98** [reply]
That repo is supposed to be fairly stable, especially the centos7 ones. Although I would suggest downloading the RPMs and not adding the '/test-plugin' repo to yum, since it is going to be abandoned soon.
